gio/src/auto/icon.rs

```diff
@@ -69,6 +63,14 @@ impl<O: IsA<Icon>> IconExt for O {
         }
     }
 
+    fn hash(&self) -> u32 {
+        unsafe {
+            ffi::g_icon_hash(
+                ToGlibPtr::<*mut ffi::GIcon>::to_glib_none(self).0 as glib::ffi::gconstpointer,
```

Comment:
Fixed by https://github.com/gtk-rs/gir/pull/1404

Reply:
Now blocked by https://github.com/gtk-rs/gtk-rs-core/pull/826 as the gir changes of that are also included